srcpkgs/cpptest/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'cpptest'
+pkgname=cpptest
+version=1.1.2
+revision=1
+build_style=gnu-configure
+build_wrksrc="$pkgname"
+hostmakedepends="autoconf automake libtool"
+short_desc="C++ Unit Testing Framework"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="LGPL-2.1-or-later"
+homepage="https://github.com/cpptest/cpptest"
+distfiles="https://github.com/${pkgname}/${pkgname}/archive/${version}.tar.gz"
+checksum=9b15cc5b08fa52eaeb5199e214a1a7c5e821240ff8514664710986cb02d60e9b
+
+
+pre_configure() {
+	./autogen.sh
+}
+
+post_install() {
+	vlicense COPYING
+}
+cpptest-devel_package() {
+	depends="$makedepends cpptest-${version}_${revision}"
+	short_desc="Cpptest - development files"
```

Comment:
you can do short_desc+=" - development files"

Reply:
Fixed, thanx :)